src/types.jl

```diff
@@ -151,41 +175,37 @@ function pgdata(::Type{PostgresType{:numeric}}, ptr::Ptr{UInt8}, data::Number)
     ptr = storestring!(ptr, string(data))
 end
 
-function pgdata(::PGStringTypes, ptr::Ptr{UInt8}, data::ByteString)
```

Comment:
@iamed2 You can see in the last [commit](https://github.com/JuliaDB/PostgreSQL.jl/pull/48/files#diff-1f61f32925620a8c37482af6f9a5d108L159) all calls to `storestring!` have been converted from `bytestring` to [`String`](https://docs.julialang.org/en/stable/stdlib/strings/#Core.String-Tuple{AbstractString}) not `unsafe_string` in order to make the call to `unsafe_convert` work.